CHANGELOG.md

```diff
@@ -0,0 +1,35 @@
+# Change Log
+
+## v0.2 (05-03-2019)
+  - Seen running on: iOS/Metal, D3D11
```

Comment:
We could probably move the supported platforms and additional crate descriptions to the README instead, and just have features/fixes here.

Reply:
Coincidentally, I added a list of platforms just now before reading this. But the idea to mention the *new* platforms in changelog, similar to how we mention new features.